samples/quickstart.js

```diff
@@ -21,10 +21,10 @@ const Logger = winston.Logger;
 const Console = winston.transports.Console;
 
 // Imports the Google Cloud client library for Winston
-const LoggingWinston = require('@google-cloud/logging-winston');
+const LoggingWinston = require('@google-cloud/logging-winston').LoggingWinston;
```

Comment:
The following needs to be updated as well:

* https://github.com/googleapis/nodejs-logging-winston/blob/typescript/samples/setup_explicit.js

Reply:
Done.